docs/source/guides/speed.rst

```diff
@@ -283,32 +318,35 @@ Check validation every n epochs
 
 **Use when:** You have a small dataset, and want to run less validation checks.
 
-You can limit validation check to only run every n epochs using the `check_val_every_n_epoch` Trainer flag.
+You can limit validation check to only run every n epochs using the ``check_val_every_n_epoch`` Trainer flag.
 
 .. testcode::
 
-    # DEFAULT
+    # default
     trainer = Trainer(check_val_every_n_epoch=1)
 
+    # runs validation after every 7th Epoch
+    trainer = Trainer(check_val_every_n_epoch=7)
 
-Set validation check frequency within 1 training epoch
-======================================================
+
+Validation within Training Epoch
+================================
 
 **Use when:** You have a large training dataset, and want to run mid-epoch validation checks.
 
 For large datasets, it's often desirable to check validation multiple times within a training loop.
-Pass in a float to check that often within 1 training epoch. Pass in an int `k` to check every `k` training batches.
-Must use an `int` if using an `IterableDataset`.
+Pass in a float to check that often within 1 training epoch. Pass in an int ``k`` to check every ``k`` training batches.
+Must use an ``int`` if using an :class:`~torch.utils.data.IterableDataset`.
 
 .. testcode::
 
-    # DEFAULT
-    trainer = Trainer(val_check_interval=0.95)
+    # default
+    trainer = Trainer(val_check_interval=1.0)
 
-    # check every .25 of an epoch
+    # check every 1/4 th of an epoch
     trainer = Trainer(val_check_interval=0.25)
 
-    # check every 100 train batches (ie: for `IterableDatasets` or fixed frequency)
```

Comment:
what's the difference between speed & debugging, given the section below?

Reply:
good point.. should be removed from here.